plugins/vpp/l3plugin/vppcalls/vpp2005/vrrp_vppcalls.go

```diff
@@ -0,0 +1,136 @@
+//  Copyright (c) 2020 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vpp2005
+
+import (
+	"fmt"
+
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2005/interface_types"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2005/ip_types"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2005/vrrp"
+	l3 "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3"
+)
+
+const (
+	centiMilliRatio uint32 = 10
+)
+
+func (h *VrrpVppHandler) vppAddDelVrrp(entry *l3.VRRPEntry, isAdd uint8) error {
+	var addrs []ip_types.Address
+	var isIpv6 bool
+	for idx, addr := range entry.IpAddresses {
+		ip, err := ipToAddress(addr)
```

Comment:
Even though most of the current code in vppcalls still uses this helper func `ipToAddress`, for new development it's better to use recently added [`ParseAddress()`](https://github.com/ligato/vpp-agent/blob/master/plugins/vpp/binapi/vpp2009/ip_types/ip_types.ba.go#L332) to convert IP from string form to [`ip_types.Address`](https://github.com/ligato/vpp-agent/blob/master/plugins/vpp/binapi/vpp2009/ip_types/ip_types.ba.go#L327) type. The old usage can be switched to this in separate PR.